src/SignalR/clients/csharp/Client.Core/src/HubConnection.cs

```diff
@@ -869,18 +869,16 @@ private async Task InvokeStreamCore(ConnectionState connectionState, string meth
         }
     }
 
-    private async Task SendHubMessage(ConnectionState connectionState, HubMessage hubMessage, CancellationToken cancellationToken = default)
+    private async Task SendHubMessage(ConnectionState connectionState, HubMessage hubMessage)
     {
         _state.AssertConnectionValid();
         _protocol.WriteMessage(hubMessage, connectionState.Connection.Transport.Output);
 
         Log.SendingMessage(_logger, hubMessage);
 
-#pragma warning disable CA2016 // Forward the 'CancellationToken' parameter to methods
         // REVIEW: If a token is passed in and is canceled during FlushAsync it seems to break .Complete()...
         await connectionState.Connection.Transport.Output.FlushAsync().ConfigureAwait(false);
```

Comment:
@BrennanConroy @halter73 ?

Reply:
Let's not make this change.

Reply:
Could you clarify - should I suppress the warning or is this change that stops passing the token fine?

Reply:
Revert the change, we obviously want to pass the token just haven't gotten around to figuring out the issue.